### ISSUE TITLE
Do you want to open a PR to merge latest into the Altium-Designer-addons repo?

### ISSUE THREAD

**Issue by RnDMonkey** [coffeenmusic/Silkscreen_Auto_Placer#3]
The version up at https://github.com/Altium-Designer-addons/scripts-libraries/tree/master/Scripts%20-%20PCB/AutoPlaceSilkscreen is out of date.

**Comment by coffeenmusic** [comment]
Thanks for reminding me. I will try and do that soon.

**Comment by RnDMonkey** [comment]
I've got my eye on this script. One of the things I do a lot is have components placed at 45° angles. Autoposition and bounding boxes really fail with these and any other angles that aren't mod 90 = 0, you know? I have some ideas on how to hack your script to work with those, if you're amenable to reviewing the changes in the main script repo later? I'd want to start from your merged PR.

**Comment by coffeenmusic** [comment]
Yeah that sounds great! That would be a very useful addition. I went ahead and got the PR submitted.
